### PR TITLE
Update v2_12_0.md

### PR DESCRIPTION
**Pull request by lparadkar-rocket**
Added notes for Zowe Application Framework.

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/getting-started/release-notes/v2_12_0.md [hunk 1]
```diff
@@ -10,10 +10,22 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
 
 Zowe Version 2.12.0 contains the enhancements that are described in the following topics.
 
-### Zowe installation and packaging
-
 ### Zowe Application Framework
 
+#### ZLUX App Server
+
+* New versions of the components can change the location of their plug-ins, as the app-server will now re-inspect the plug-in locations on each startup.( [#280](https://github.com/zowe/zlux-app-server/pull/280))
```
Comment (anaxceron) [inline docs/getting-started/release-notes/v2_12_0.md:17]: please fix the spacing in your link

File: docs/getting-started/release-notes/v2_12_0.md [hunk 2]
```diff
@@ -10,10 +10,22 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
...
+
+#### ZLUX Server Framework
+
+* Auth plugins that are not requested by any dataservice found at startup are no longer loaded by the server. ([#504](https://github.com/zowe/zlux-server-framework/pull/504))
```
Comment (anaxceron) [inline docs/getting-started/release-notes/v2_12_0.md:21]: use a hyphen in _plug-ins_, as per our style guide

File: docs/getting-started/release-notes/v2_12_0.md [hunk 3]
```diff
@@ -10,10 +10,22 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
...
+
+#### ZSS
+
+* ZSS now defaults to using `TLS 1.3`, which requires a minimum of zOS Version `2.4` ([#646](https://github.com/zowe/zss/pull/646))
```
Comment (anaxceron) [inline docs/getting-started/release-notes/v2_12_0.md:25]: Sentence should end w/ a period

File: docs/getting-started/release-notes/v2_12_0.md [hunk 4]
```diff
@@ -10,10 +10,22 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
...
+* Added configuration parameter `components.zss.agent.https.maxTls` to control which level of TLS to use, allowing downgrading to `TLS 1.2` if desired, with value `TLSv1.2` ([#654]( https://github.com/zowe/zss/pull/654))
```
Comment (anaxceron) [inline docs/getting-started/release-notes/v2_12_0.md:26]: End sentence w/ a period

File: docs/getting-started/release-notes/v2_12_0.md [hunk 5]
```diff
@@ -29,8 +41,14 @@ Zowe Version 2.12.0 contains the bug fixes that are described in the following t
 
 ### Zowe installation and packaging
 
+* Fixed a rare bug where zwe could hang when running a unix program ([#3590](https://github.com/zowe/zowe-install-packaging/pull/3590))
```
Comment (anaxceron) [inline docs/getting-started/release-notes/v2_12_0.md:44]: End the sentence w/ a period

File: docs/getting-started/release-notes/v2_12_0.md [hunk 6]
```diff
@@ -29,8 +41,14 @@ Zowe Version 2.12.0 contains the bug fixes that are described in the following t
...
+
 ### Zowe Application Framework
 
+#### ZLUX App Server
+
+* Removed error message `components/app-server/bin/configure.sh 26: .: FSUM6807 expression syntax error` seen in startup of Zowe in  Version `2.11.0`, caused by incorrect shell syntax. ( [#280](https://github.com/zowe/zlux-app-server/pull/283))
```
Comment (anaxceron) [inline docs/getting-started/release-notes/v2_12_0.md:50]: Please fix the spacing in your parenthetical link

**Review by anaxceron — APPROVED** [review]

**Comment by lparadkar-rocket** [comment]
This was never merged and not needed now.

**Pull request closed without merging**